Editor.js

```diff
@@ -30,6 +31,16 @@ define([
 				self._focusedEditorCell = null;
 			});
 			this._listeners.push(this._editorFocusoutHandle);
+
+			aspect.before(this, 'refresh', function() {
+				var id;
+				for (id in this._editorInstances) {
+					var editorInstanceDomNode;
```

Comment:
This would be more easily readable if you just perform the assignment here rather than in the loop.  Why wait until the next statement?